src/main/package/jpackage-ubuntu-latest.cfg

```diff
@@ -0,0 +1,5 @@
+--type deb
+--arguments "-nolf"
+--icon src/main/package/logo.ico
```

Comment:
maybe a .png would be better for linux...

Reply:
good, I changed that and added a png icon